csaf_files/IT/white/2024/va-24-325-01.json

```diff
@@ -0,0 +1,195 @@
+          "title": "Description"
+        },
+        {
+          "category": "details",
+          "title": "SSVC",
+          "text": "SSVCv2/E:P/A:Y/T:T/2024-11-20T18:23:05.686749+00:00/"
+        }
+      ],
+      "title": "Versa Networks Versa Director insecure default PostgreSQL configuration",
+      "product_status": {
+        "known_affected": [
+          "CSAFPID-0002"
+        ],
+        "fixed": [
+          "CSAFPID-0001"
+        ]
+      },
+      "references": [
+        {
+          "category": "external",
+          "summary": "security-portal.versa-networks.com",
+          "url": "https://security-portal.versa-networks.com/emailbulletins/6735a300415abb89e9a8a9d3"
+        },
+        {
+          "category": "external",
+          "summary": "docs.versa-networks.com",
+          "url": "https://docs.versa-networks.com/Solutions/System_Hardening/Perform_Manual_Hardening_for_Versa_Director#Secure_HA_Ports"
+        }
+      ],
+      "scores": [
+        {
+          "cvss_v3": {
+            "baseScore": 10,
+            "baseSeverity": "CRITICAL",
+            "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:C/C:H/I:H/A:H",
+            "version": "3.1"
+          },
+          "products": [
+            "CSAFPID-0002"
+          ]
+        }
+      ],
+      "remediations": [
+        {
+          "category": "vendor_fix",
+          "details": "From: [Advising Vulnerability In Versa Director](https://security-portal.versa-networks.com/emailbulletins/6735a300415abb89e9a8a9d3): \"Starting with the latest 22.1.4 version of Versa Director, the software will automatically restrict access to the Postgres and HA ports to only the local and peer Versa Directors.\" The advisory does not directly address the \"...common password across all instances of Versa Director.\"",
+          "url": "https://security-portal.versa-networks.com/emailbulletins/6735a300415abb89e9a8a9d3",
+          "product_ids": [
+            "CSAFPID-0001"
+          ],
+          "date": "2024-11-11T06:00:00.000Z"
+        },
+        {
+          "category": "vendor_fix",
+          "details": "From: [Advising Vulnerability In Versa Director](https://security-portal.versa-networks.com/emailbulletins/6735a300415abb89e9a8a9d3): \"Starting with the latest 22.1.4 version of Versa Director, the software will automatically restrict access to the Postgres and HA ports to only the local and peer Versa Directors.\" The advisory does not directly address the \"...common password across all instances of Versa Director.\"",
+          "url": "https://security-portal.versa-networks.com/emailbulletins/6735a300415abb89e9a8a9d3",
+          "product_ids": [
+            "CSAFPID-0002"
+          ],
+          "date": "2024-11-11T06:00:00.000Z"
+        }
+      ],
+      "acknowledgments": [
+        {
+          "names": [
+            "[notnotnotveg](https://wiki.notveg.ninja/whoami/)"
+          ]
+        }
+      ],
+      "release_date": "2024-11-14T00:00:00"
```

Comment:
The date-time format recommends including the "Z" at the end of the timestamp.

Reply:
Done.

Reply:
I checked and some of the older files are mising the 'Z' on release_date. Editing the JSON files in the PR will break the signatures and hashes so I'll update the PR.